function-images/tests/video-bench/object_recognition/recog.py

```diff
@@ -0,0 +1,120 @@
+"""Azure Function to perform inference.
+"""
+from torchvision import transforms
+from PIL import Image
+import torch
+import torchvision.models as models
+
+import sys
+import os
+# adding python tracing sources to the system path
+sys.path.insert(0, os.getcwd() + '/../proto/')
+import videoservice_pb2_grpc
+import videoservice_pb2
+
+import io
+import grpc
+import argparse
+
+from concurrent import futures
+from timeit import default_timer as now
+
+start_overall = now()
+
+parser = argparse.ArgumentParser()
+parser.add_argument("-sp", "--sp", dest = "sp", default = "80", help="serve port")
+
+args = parser.parse_args()
+
+# DMITRII: The code below should run upon initialization
+
+# Load model
+#model_file = 'squeezenet.pt'
+start_load_model = now()
+#model = torch.load(model_file)
+model = models.squeezenet1_1(pretrained=True)
+#mode = 
+end_load_model = now()
+load_model_e2e = (end_load_model - start_load_model) * 1000
+print('Time to load model: %dms' % load_model_e2e)
+
+labels_fd = open('imagenet_labels.txt', 'r')
+labels = []
+for i in labels_fd:
+    labels.append(i)
+labels_fd.close()
+
+# Starting from here, the code should run upon a function invocation
+
+def processImage(bytes):
+    start_preprocess = now()
+    # Load image
+    # img = Image.open('frame10.jpg')
+    img = Image.open(io.BytesIO(bytes))
```

Comment:
You should take it from grpc!

Reply:
`bytes` comes from the grpc request already, io.BytesIO is just an interface.